apps/engineeringblog/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "next/core-web-vitals"
```

Comment:
See other apps e.g. RoboShield for what needs to go here... basically I think we need at least `prettier` for formatting.